src/main/java/net/dv8tion/jda/api/entities/MessageSticker.java

```diff
@@ -114,6 +115,19 @@ public String getIconUrl()
         return Helpers.format(ICON_URL, getId(), formatType.getExtension());
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this sticker's image
+     *
+     * @return Never-null {@link ImageProxy} of this sticker's image
+     *
+     * @see    #getIconUrl()
+     */
+    @Nonnull
+    public ImageProxy getIcon()
+    {
+        return new ImageProxy(getIconUrl());
```

Comment:
`getIconUrl` has a documented exception that should also be documented on this method